sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/source/ContainersMetadataTopicPartition.java

```diff
@@ -12,22 +12,30 @@
 
 public class ContainersMetadataTopicPartition {
     public static final String DATABASE_NAME_KEY = "database";
+    public static final String CONNECTOR_NAME_KEY = "connectorName";
 
     private final String databaseName;
+    private final String connectorName;
 
-    public ContainersMetadataTopicPartition(String databaseName) {
+    public ContainersMetadataTopicPartition(String databaseName, String connectorName) {
         checkArgument(StringUtils.isNotEmpty(databaseName), "Argument 'databaseName' can not be null");
 
         this.databaseName = databaseName;
+        this.connectorName = StringUtils.isEmpty(connectorName) ? StringUtils.EMPTY : connectorName;
```

Comment:
Shouldn't we have a good default value?

Reply:
want to keep this way for now to show that there is no connector name being configured.